### ISSUE TITLE
Dark mode/themes for epub reader

### ISSUE THREAD

**Issue by hotsno** [janeczku/calibre-web#1960]
**Is your feature request related to a problem? Please describe.**
My eyes burn when reading using the epub reader (epub.js) in the dark.

**Describe the solution you'd like**
Option to change themes in the epub reader.

**Describe alternatives you've considered**
I was thinking about trying to do this myself, but as I have next to no experience in contributing to projects and am still a noob coder so I figured I should just make a feature request. I'll still probably try.

**Additional context**
Some people have already tried this. Here are some helpful links to those Issues:
https://github.com/janeczku/calibre-web/issues/778#issuecomment-456560756
https://github.com/janeczku/calibre-web/pull/1736
https://github.com/janeczku/calibre-web/pull/1796

**Comment by WingsZeng** [comment]
Hi, there! I noticed that the author seems not to support dark mode or theme for epub reader officially, because there's no reason to edit `reader.min.js` (or something else). So I wrote a dark mode switch like a patch, which can be easily installed without lots conflicts. Here's the [repository](https://github.com/WingsZeng/cw-dark-epub) and here's the screenshot:

![](https://github.com/WingsZeng/cw-dark-epub/raw/master/screenshot/dark.png)
![](https://github.com/WingsZeng/cw-dark-epub/raw/master/screenshot/light.png)

Also the PR #2223 works, but I prefer a simple dark mode switch. :) I also changed colors in sidebar to meet dark mode, which is not support by PR #2223 .

Hope this will help you!
